app/grandchallenge/participants/views.py

```diff
@@ -107,3 +111,74 @@ def get_success_url(self):
             "participants:registration-list",
             kwargs={"challenge_short_name": self.object.challenge.short_name},
         )
+
+
+class RegistrationQuestionList(
+    LoginRequiredMixin, ObjectPermissionRequiredMixin, ListView
+):
+    model = RegistrationQuestion
+    permission_required = "change_challenge"
```

Comment:
It is safer to use the existing `view_registrationquestion` object permission, and assign this to the challenge editors group when the question is created.